Examples/ByFeatures/DependencyInjection/FeatureFixture.fs

```diff
@@ -0,0 +1,23 @@
+﻿module TickSpec.NUnit
+
+open System.Reflection
+open NUnit.Framework
+open TickSpec
+
+let assembly = Assembly.GetExecutingAssembly() 
```

Comment:
consider separating this out like in xUnit2 with a `TickScenarios.fs` (that a NuGet package could emit - ideally with a type that you don't have to derive from and can just use as a one line) and a `Features.fs` which in one line lets you expose a set of scenarios to the runner. (Only if you have appetite, this is unimportant and I will prob do it as a backport as part of the work of porting your example to xUnit -- but it is nice to have a new user not have a mangle of tech code with no comments and a base class they need to understand)